package.json

```diff
@@ -42,7 +42,6 @@
     "test:e2e:saucelabs": "wdio wdio.saucelabs.conf.js",
     "test:size": "bundlesize",
     "test:argos": "argos upload functional-tests/screenshots --token $ARGOS_TOKEN || true",
-    "release": "./scripts/release/publish.js",
     "release:maintenance": "./scripts/release/publish-maintenance.sh",
     "release:view-changelog": "conventional-changelog -p angular",
```

Comment:
We actually can remove these.
When we want to release the old versions like v3, we will checkout v3 and there we'll have publish-maintenance.sh file :)

Reply:
scripts/release/bump-package-version.js
scripts/release/conventional-changelog.js
scripts/release/publish-maintenance.js
scripts/release/publish.js